source/integrate-with-integration-environment/authenticate-your-user.html.md.erb

```diff
@@ -202,7 +202,6 @@ Common reasons for the error include:
 * the `redirect_uri` parameter does not match one of the uris configured in your client configuration, this is a particularly common cause of this error and it is important that the match is exact
 * the `response_type` parameter is not set to `code`
 * the `scope` parameter is incorrect
```

Comment:
This could be more specific.
`Internal server error` is returned if `openid` is missing from the requested scopes. If `openid` is present but other incorrect scopes are also present then the `invalid_scope` is returned instead

Reply:
Good point, I'll tweak the working and push the change up